### PR TITLE
Feature/cz divider modifier

### PR DESCRIPTION
**Pull request by nicoes**
A refactor of the cz-divider after discussing with @hvgeertruy . We now have two independent components 'cz-divider-timer' and 'cz-divider' (that do have a dependency on a shared mixin). This also means that dividers are decoupled from cz-headings (which was not the case).

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@ <h2 class="cz-heading-1-sub">Interactieve, gebruiksvriendelijke (web)applicaties
 
             <div class="media-block__content">
                 <h2>Onze visie</h2>
+                <div class="cz-divider"></div>
```
Comment (nicoes) [inline content/index.html]: True!

File: src/scss/components/divider-timer.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+@import 'scss/tools/divider';
+@import 'scss/settings/variables';
+
+.cz-divider-timer {
+    @include cz-divider;
+    position: relative;
+    background: #fff;
```
Comment (nicoes) [inline src/scss/components/divider-timer.scss]: Okay

File: src/scss/components/divider-timer.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    overflow: hidden;
+
+    @media screen and (min-width: $cz-breakpoint-desktop) {
```
Comment (nicoes) [inline src/scss/components/divider-timer.scss]: I've added this.

**Comment by hvgeertruy** [comment]
hmm not sure why this happens:

Build failing:
Error: File to import not found or unreadable: node_modules/normalize-css/normalize.
Parent style sheet: /home/travis/build/codezilla-nl/codezilla-website/src/main.scss -> /home/travis/build/codezilla-nl/codezilla-website/src/main.scss:14:1

It should work (working locally / other builds)

**Pull request merged**